docs/changelogs/v2.10.0.md

```diff
@@ -0,0 +1,125 @@
+
+
+## Changelog
+
+> [!NOTE]
+> This is a Mainline Coder release. We advise enterprise customers without a staging environment to install our [latest stable release](https://github.com/coder/coder/releases/latest) while we refine this version. Learn more about how we release in our [docs](../install/releases.md).
```

Comment:
Do we want to capitalize mainline?  My preference is lowercase 
<img width="866" alt="Screenshot 2024-04-03 at 3 05 20 PM" src="https://github.com/coder/coder/assets/22407953/3627e1d1-8c91-427a-95ba-6a8668e28740">

Reply:
Fixed, lowercase only.